setup.py

```diff
@@ -0,0 +1,15 @@
+import setuptools
+
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(name='hsl_builder',
+      version='0.1',
+      description='HSL Builder for creating hsl elements',
+      license='MIT',
```

Comment:
Let's confirm this with devops team first.

Reply:
ok :+1:  
P.S I had used this following Relements which also uses MIT

Reply:
Got it.